### ISSUE TITLE
Show links to download Clients after signing up

### ISSUE THREAD

**Issue by jamilbk** [CloudFire-LLC/cloudfire#5678]
- [ ] Show in view after creating account
- [ ] Show link to Quickstart guide
- [ ] Show in welcome email

**Comment by AndrewDryga** [comment]
Should we do an onboarding widget that will show X steps you need to do to finish trying Firezone? Like:

- [ ] Create a Resource
- [ ] Create a Policy
- [ ] Deploy a Gateway
- [ ] Connect a Client

and they will be checked out automatically when at least one of given entities exist for an account, completely hidden after all are checked out.

**Comment by jamilbk** [comment]
Yeah that's not a bad idea. Where are you thinking? Below the breadcrumbs?
